assets/less/hel/hearing-page.less

```diff
@@ -228,7 +228,6 @@
         font-size: 20px;
     }
     .commentlist {
```

Comment:
Oh, the css implementation requires moving the sort-selector after the CommentForm element, but no other changes needed.